src/cmdline.rs

```diff
@@ -2,262 +2,261 @@ use clap::{App, Arg,  AppSettings};
 use crate::version;
 
 pub fn parse<'a>() -> clap::ArgMatches<'a> {
-  let matches = create_expected_args().get_matches();
-  matches
+	let matches = create_expected_args().get_matches();
+	matches
 }
 
 pub fn create_expected_args<'a>() -> App<'a, 'a> {
-  App::new("fwumious wabbit")
```

Comment:
a bit hard to evaluate the change with this indentation - I suggest to separate purely indentation from code changes to make it easier to find the relevant changes